packages/bot-web-ui/src/assets/sass/load-modal.scss

```diff
@@ -0,0 +1,290 @@
+$modal_container_height: calc(100vh - 210px);
+
+.load {
+    &__container {
+        padding: 15px 5px;
+    }
+    &__content-with-footer {
+        max-height: calc(100vh - 280px);
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.recent {
```

Comment:
Could you scope this under `.load`? It'll prevent collision with other elements that have the same class.